example/LockFreeQueue.chpl

```diff
@@ -0,0 +1,98 @@
+/**
+ * Created by Garvit Dewan - 
+ * https://github.com/dgarvit/epoch-based-manager/blob/master/src/LockFreeQueue.chpl
+ *   
+ * Lock-Free Queue that uses ABA feature of Distributed Data Structures
```

Comment:
Should probably say that it uses the ABA feature of `LocalAtomicObject`. Sure the `LocalAtomicObject` was created back in GSoC 2017, but you don't need to state that here.

Reply:
Fixed it.